### ISSUE TITLE
Compatible node version?

### ISSUE THREAD

**Issue by sdevilliers** [vyakymenko/angular-seed-express#246]
[ x] bug report => search github for a similar issue or PR before submitting

**Current behavior**
Download the current version of seed, do the npm install
- npm run build.prod
- npm run serve.prod
Currently the webpage opens with the page "Loading..."
No errors but the bootstrapping does not appear to be working.

**Expected behavior**
The sample page with the scientists should come up.

**Minimal reproduction of the problem with instructions**
As above

**Please tell us about your environment:**
Mac OS 10.14.2
Webstorm 18.3
npm 6.9.0

* **Angular Seed Version:** `ea47591`
* **Node:** `node --version` = I have tried multiple versions here with different behavior.
I have tried 6.9.0 & 10.15.3 & 11.12.0 to name a few.
Does not want to work for me.
Would welcome a pointer in the right direction.
Thanks!

**Comment by vyakymenko** [comment]
Hello @sdevilliers , 

My config:
![Screenshot at Mar 17 09-38-33](https://user-images.githubusercontent.com/7300673/54487031-19b50280-4899-11e9-9189-975b62addb68.png)

After `$ npm install` I ran `$ npm start` and everything works fine.

![Screenshot at Mar 17 09-41-31](https://user-images.githubusercontent.com/7300673/54487033-276a8800-4899-11e9-87e0-dd4af73b78dc.png)


Thanks!

**Comment by sdevilliers** [comment]
Um, Yes. I did manage to get the dev mode working also.
But I was referring to the **prod mode** setup.

I followed your steps as closely as possible as follows:
Used "n" to install V10.15.0 of node - to match your version.
Downloaded a fresh copy of the code - git clone http: vyakymenko etc
(npm version **6.9.0** one minor higher) was used to "npm install"
Followed by an npm start - This works as you suggested above.

**npm run serve.prod** still does not work for me. No errors in the console.
![image](https://user-images.githubusercontent.com/45904117/54498209-c1e2bf80-48c9-11e9-9291-e4de36fc1a81.png)

![image](https://user-images.githubusercontent.com/45904117/54498247-6b29b580-48ca-11e9-9e05-7b9d087f3b31.png)
